modules/core/src/main/scala/ch/epfl/scala/debugadapter/internal/stepfilter/ClassLoadingFilter.scala

```diff
@@ -2,8 +2,11 @@ package ch.epfl.scala.debugadapter.internal.stepfilter
 
 import com.sun.jdi.{Location, Method}
 import ch.epfl.scala.debugadapter.internal.ByteCodes
+import java.util.Optional
 
 object ClassLoadingStepFilter extends StepFilter {
+  override def formatName(method: Method): Optional[String] = Optional.of(method.name())
```

Comment:
Maybe `format` is a better name for this method. We don't format only the name of the method but also its arguments and return type.

Reply:
Should we rename `StepFilter` into `StackFrameHandler`?